src/components/TodoFilter/TodoFilter.tsx

```diff
@@ -1,34 +1,75 @@
-export const TodoFilter = () => (
-  <form className="field has-addons">
-    <p className="control">
-      <span className="select">
-        <select data-cy="statusSelect">
-          <option value="all">All</option>
-          <option value="active">Active</option>
-          <option value="completed">Completed</option>
-        </select>
-      </span>
-    </p>
+import { FilterType } from '../../types/FilterType';
 
-    <p className="control is-expanded has-icons-left has-icons-right">
-      <input
-        data-cy="searchInput"
-        type="text"
-        className="input"
-        placeholder="Search..."
-      />
-      <span className="icon is-left">
-        <i className="fas fa-magnifying-glass" />
-      </span>
+type Props = {
+  query: string
+  setQuery: (input: string) => void
+  filterType: FilterType
+  setFilterType: (value: FilterType) => void
+};
 
-      <span className="icon is-right" style={{ pointerEvents: 'all' }}>
-        {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-        <button
-          data-cy="clearSearchButton"
-          type="button"
-          className="delete"
+export const TodoFilter: React.FC<Props> = ({
+  query,
+  setQuery,
+  filterType,
+  setFilterType,
+}) => {
+  const handleSelection = (event: React.ChangeEvent<HTMLSelectElement>) => {
+    switch (event.target.value) {
+      case FilterType.Active:
+        setFilterType(FilterType.Active);
+        break;
+
+      case FilterType.Completed:
+        setFilterType(FilterType.Completed);
+        break;
+
+      default:
+        setFilterType(FilterType.All);
+        break;
+    }
+  };
```

Comment:
I think it would be clearer and less code if you cleared the switch case and called setFilterType with event.target.value.